guides/GITHUB.md

```diff
@@ -0,0 +1,334 @@
+directly from where they live. You can read about this feature, called [GitHub Pages](https://pages.github.com/).
+
+To summarize and answer this question, we use GitHub because we want to:
+
+ 1. easily collaborate on common files
+ 2. have advanced reviewing tools and version history
+ 3. keep a record of discussions, and every change made
+ 4. be able to revert back to previous versions, if needed
+ 5. render some of the static files in a web interface
+
+
+## What content do we have on GitHub?
+
+Generally, anything that you'd want to put in a digital safe deposit box is what you'll
+find on GitHub. This obviously includes code, but also includes documentation and
+other web content. Specifically, the [sourcecred organization](https://github.com/sourcecred)
+has the following sets of files. Each set is called a "repository" and you can think of
+it like a (sub)-project under a GitHub organization, or a GitHub username. Each repository
+has it's own link and space on the GitHub website.
+
+ - [sourcecred](https://github.com/sourcecred/sourcecred): the core sourcecred codebase.
+ - [docs](https://github.com/sourcecred/docs): is a growing set of documentation and community health files. In fact, the file that you are reading right now is part of it!
+ - [sourcecred-action](https://github.com/sourcecred/sourcecred-action): a set of workflow that can be used by other GitHub communities.
+
+If you are familiar with filesystems and permissions, you can think of each
+repository as a folder or space that carries it's own permissions, labels,
+and sets of issues, pull requests (requests for changes), milestones, and sometimes
+even guides for contribution. Each is an independent entity under the organization, and thus each
+carries it's own kind of expertise. For example, 
+if you like working with configuration files and workflows, `sourcecred-action`
+might be for you. If you aren't a programmer, you might be most comfortable starting
+with `docs`.
+
+
+### Is my help needed in a software project when I'm not a programmer?
+
+Yes! Any substantial software project is multi-disciplinary. This one included. In fact, calling SourceCred a software project doesn't do justice to how much work besides programming goes into it.
+
+
+### Why and how should I get involved here specifically?
+
+It's easy to think:
+
+> GitHub seems like a platform for programmers. I'm not a programmer, why would anyone want my help?
+
+But here's the thing! When you look deeper at a lot of discussions
+that are had around GitHub repositories, while some of them might require code 
+expertise, a large portion do not. Even when the changes are specifically to
+suggest changes to code, it's usually the responsibility of the individual
+asking for feedback to explain their changes in human friendly terms. For
+example, a developer might say something like:
+
+> This (review request) is going to add a plugin to support reddit as a place to get cred.
```

Comment:
To me, the purpose of this part is to say, "Here's some tech jargon that might be said that you might not understand, and here's how you could contribute regardless of not understanding it." I think that's great. However, I think the action points after the jargon could be clearer. As a beginner, I would read the jargon, not get it, and then not understand the action points presented. The questions beginners would be asking would probably be a bit less complex.

Reply:
I cleaned this up locally - will push with this round of changes after we discuss the open ended points!